organizations/backends.py

```diff
@@ -34,11 +50,37 @@ def authenticate(self, *args, **kwargs):
         # superuser can log into any microsite
         site = get_current_site()
         is_default_site = site.id == settings.SITE_ID
+
         if not is_default_site and is_site_configuration_enabled() and user and not user.is_superuser:
```

Comment:
@OmarIthawi or @bryanlandia I know this is code not modified in this PR and also this line here is original code from Filip, but do you know why we need to make the call to `is_site_configuration_enabled{}` ? 

When would a site NOT have a "configuration" attribute?

Here's the source and "blame" for the `is_site_configuration_enabled` function:
The code is from upstream: https://github.com/appsembler/edx-platform/blame/appsembler/tahoe/master/openedx/core/djangoapps/site_configuration/helpers.py#L30-L41

Why could we not check if there is an `organizations.models.UserOrganizationMapping` model instead?

Reply:
I don't really know.